_sass/_Footer.sass

```diff
@@ -33,9 +33,9 @@ footer
         line-height: 4rem
 
       a.logo
-        background: url(https://assets.openshift.net/content/subdomain/redhat_reverse.png) no-repeat
-        background-size: 93.4px 30px
+        background: url(./redHatLogo.png) no-repeat
+        background-size: 144px 30px
```

Comment:
I think the logo will look better if we change it to:
```suggestion
        background-size: 144px 34px
```
(maintaining aspect ratio)

Before:
<img width="479" alt="Screen Shot 2020-03-31 at 09 45 10" src="https://user-images.githubusercontent.com/1079279/78027854-706e6c80-7334-11ea-99e3-eb47a0968f1f.png">

After:
<img width="438" alt="Screen Shot 2020-03-31 at 09 45 18" src="https://user-images.githubusercontent.com/1079279/78027882-7cf2c500-7334-11ea-9fd3-28e5db892291.png">